content/sensu-go/6.0/reference/filters.md

```diff
@@ -832,6 +832,97 @@ spec:
 
 {{< /language-toggle >}}
 
+## Use built-in JavaScript filter execution functions
+
+Sensu includes four built-in JavaScript functions for event filter execution:
+
+- `sensu.CheckDependencies`
+- `sensu.FetchEvent`
+- `sensu.CheckStatus`
+- `sensu.ListEvents`
+
+You can use these functions to query your event stores.
+
+### `sensu.CheckDependencies`
+
+The `sensu.CheckDependencies` function takes zero or more checks as arguments.
+It returns `true` if all the specified checks are passing or `false` if any of the specified checks are failing.
+If you do not specify any checks, the function returns `true`.
+
+You can refer to the checks as strings:
+
+{{< code javascript >}}
+sensu.CheckDependencies("database", "disk")
+{{< /code >}}
+
+If you pass the check names as strings, Sensu assumes that the entites are the same as those in the events being filtered.
+
+You can also refer to the checks in objects that include both the entity and check name.
+For example:
+
+{{< code javascript >}}
+sensu.CheckDependencies({entity: "server01", check: "disk"}, {entity: "server01", check: "database"})
+{{< /code >}}
+
+In both cases, if no event matches the specified entities and checks, Sensu will raise an error.
+
+### `sensu.EventStatus`
+
+The `sensu.EventStatus` function takes zero or more checks as arguments.
```

Comment:
See the examples for the spec of what the function takes. It can either take string check names, or objects that specify an entity name and check name. If only the check name is given, then the entity is computed from the event that is being filtered.

Reply:
I figured that but the gist just said it takes the same arguments as sensu.CheckDependencies so I wanted to make sure.